File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int a, int b) {
    int answer = 0;
    //
    int c = stoi(to_string(a) + to_string(b));
    answer = max(c, 2 * a * b);
    //
    return answer;
}

File: 프로그래머스/0/181938. 두 수의 연산값 비교하기/두 수의 연산값 비교하기.java
import java.util.*;

class Solution {
    public int solution(int a, int b) {
        int answer = 0;

        if (Integer.valueOf(String.valueOf(a) + String.valueOf(b)) > 2 * a * b) {
            answer = Integer.valueOf(String.valueOf(a) + String.valueOf(b));
        } else {
            answer = 2 * a * b;
        }

        return answer;
    }
}